lib/ember-test-helpers/test-module.js

```diff
@@ -133,6 +133,7 @@ export default Klass.extend({
 
     setContext({
       container:  this.container,
+      registry: this.registry,
```

Comment:
Exposing the registry to the text context lets people move to the non-deprecated registration APIs. This starts giving us a path away from the hacks that are currently used to avoid deprecations.